docs-2.0/5.configurations-and-logs/1.configurations/1.get-configurations.md

```diff
@@ -0,0 +1,34 @@
+# Get configurations
+
+This document gives two methods to get configurations in Nebula Graph.
+
+> **DON'T:** DO NOT use the two methods at one time.
+
+To avoid errors, we suggest that you use the first method.
```

Comment:
What kind of errors would the second method cause?  Maybe provide a brief description so that users can evaluate the situation when they feel like using the 2nd method.